src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -150,16 +173,16 @@ private IEnumerable<string> FindFiles(string[] includePatterns, string[] exclude
             }
 
             var matches = matcher.Execute(_baseGlobbingDirectory);
+            var matchedUrls = new List<string>();
+            foreach (var matchedPath in matches.Files)
+            {
+                // Resolve the path to site root
+                var relativePath = new PathString("/" + matchedPath.Path);
+                matchedUrls.Add(RequestPathBase.Add(relativePath).ToString());
```

Comment:
Should we do `basepath + '/' + matchedPath.Path` instead? This is 2 concats as-written.